src/Appender/AppenderSkeleton.cs

```diff
@@ -321,7 +321,7 @@ public void DoAppend(LoggingEvent loggingEvent)
 				{
 					ErrorHandler.Error("Failed in DoAppend", ex);
 				}
-#if !MONO && !NET_2_0 && !NETSTANDARD1_3
+#if !(MONO || NET_2_0 || NETSTANDARD1_3 || NETSTANDARD2_0)
```

Comment:
I used the FrameworkMajor_Minor convention because it's used by the build system, e.g. `NETSTANDARD1_3` is defined when building a `netstandard1.3` library (more details in the Target Frameworks [doc](https://docs.microsoft.com/en-us/dotnet/standard/frameworks)).